glusterd2/commands/devices/transaction.go

```diff
@@ -0,0 +1,35 @@
+package devicecommands
+
+import (
+	"os/exec"
+	"strings"
+
+	device "github.com/gluster/glusterd2/glusterd2/device"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	log "github.com/sirupsen/logrus"
+)
+
+func txnPrepareDevice(c transaction.TxnCtx) error {
+	var deviceinfo device.Info
```

Comment:
Use the transaction logger consistently

Reply:
@prashanthpai  
so are you suggesting to use c.Ctx.Logger() ?

Reply:
Yes. For example, refer to step function of other transactions in the repo.